code/modules/mob/living/carbon/combat.dm

```diff
@@ -115,6 +115,12 @@
 		return
 	if(restrained() || lying || locked_to || stat)
 		return
+	if(src.mind.special_role == BOMBERMAN)
```

Comment:
No, that's not the right way to do it. I can accept blocking them, I am not really fine with gibbing them for not knowing it's not possible. But then again, it's just Bomberman.

Reply:
[https://github.com/vgstation-coders/vgstation13/blob/3474e398173fca3a0b4e06c665b5919a792bfa22/code/modules/mob/living/carbon/human/human_attackhand.dm#L252](url)
It's handled the same way with disarming, the only thing I added was the farting sfx.